.rubocop.yml

```diff
@@ -149,6 +149,9 @@ MethodLength:
 ModuleFunction:
   Enabled: false
 
+MutableConstant:
+  Enabled: false
```

Comment:
I revised my commit and enabled to true.  As a consequence it raised about 8 errors for the constants, so I went ahead and froze them all and added the frozen_string_literal comment for the affected files.  This appears to be the only practical way to take advantage of frozen strings until 3.0.